modules/cardano-service.nix

```diff
@@ -4,6 +4,7 @@ globals: params:
 { config, pkgs, lib, options, ... }:
 let
   cfg = config.services.cardano-node;
+  assetLockFile = pkgs.writeText "asset-lock-file" (intersperse "\n" cfg.assetLockAddresses);
```

Comment:
Can the asset lock file be provided directly as a `.txt` file (`../static/asset-lock-file.txt`) rather than a nix expression?